src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Connect.cs

```diff
@@ -52,7 +52,7 @@ public async Task ConnectMethod_Success()
 
                             TextReader clientReader = new StreamReader(clientStream);
                             TextWriter clientWriter = new StreamWriter(clientStream) { AutoFlush = true };
-                            TextWriter serverWriter = connection.Writer;
+                            TextWriter serverWriter = new StreamWriter(connection.Stream) { AutoFlush = true };
```

Comment:
Nit: I don't think it'll affect anything, especially since the serverWriter isn't being disposed, but consider making leaveOpen: true, since the serverWriter doesn't actually have ownership of the stream.